lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -130,6 +130,13 @@ def expected_path(constant)
         end
 
         def camel_to_snake_case(string)
+          if defined?(ActiveSupport::Inflector)
+            if File.exist?('./config/initializers/inflections.rb')
+              require './config/initializers/inflections'
+            end
```

Comment:
Inflection configuration must be loaded before `ActiveSupport::Inflector.underscore` to take acronyms and other inflection rules into account.

That configuration is typically found in the `config/initializers/inflections.rb` file that is generated by default in new Rails apps.